### PR TITLE
Szal 14 changes to footer with map methos and css

### PR DESCRIPTION
**Pull request by ValentinoMaciej**

### CODE REVIEW

File: fitpals/src/components/FindPal/FindPal.jsx [hunk 1]
```diff
@@ -1,3 +1,9 @@
+import { Search } from "./Search/Search";
+
 export const FindPal = () => {
-  return <p>znajdzfitpala</p>;
+  return (
+    <div>
+      <Search />
+    </div>
```
Comment (Patrex97) [inline fitpals/src/components/FindPal/FindPal.jsx:7]: remove `<div>`

File: fitpals/src/components/FindPal/Search/Search.module.css [hunk 2]
```diff
@@ -0,0 +1,59 @@
...
+  justify-content: center;
+  align-items: center;
+  flex-direction: column;
+}
+.forms {
+  display: flex;
+  justify-content: center;
+  align-items: center;
+  flex-direction: column;
+}
```
Comment (Patrex97) [inline fitpals/src/components/FindPal/Search/Search.module.css:45]: `selectionInpu`t and `forms` shares same styles

File: fitpals/src/components/Footer/Footer.module.css [hunk 3]
```diff
@@ -58,8 +51,21 @@
...
+}
 
+.DevList {
+  display: flex;
   justify-content: center;
+  align-items: center;
+  flex-direction: row;
+}
 
+.devContainer {
```
Comment (Patrex97) [inline fitpals/src/components/Footer/Footer.module.css:65]: Use only one naming convention PascalCase or camelCase

File: fitpals/src/components/SearchForm/SearchForm.jsx [hunk 4]
```diff
@@ -1,3 +1,12 @@
...
 export const SearchForm = () => {
-  return <p>search form</p>;
+  return (
+  <>
+  <form>  
+  Select Start and End Date: <br><br>  
+    <input type="date" name="Startdate"> Start date:<br><br>  
+    <input type="date" name="Enddate"> End date:<br><br>  
+   <input type="submit">  
+</form>  </> )
```
Comment (Patrex97) [inline fitpals/src/components/SearchForm/SearchForm.jsx:11]: remove `<>` tag

**Review by Patrex97 — CHANGES_REQUESTED** [review]

File: fitpals/src/components/FindPal/FindPal.jsx (hunk 1 above)
Comment (ValentinoMaciej) [reply fitpals/src/components/FindPal/FindPal.jsx]: done

File: fitpals/src/components/FindPal/Search/Search.module.css (hunk 2 above)
Comment (ValentinoMaciej) [reply fitpals/src/components/FindPal/Search/Search.module.css:40]: done

File: fitpals/src/components/SearchForm/SearchForm.jsx (hunk 4 above)
Comment (ValentinoMaciej) [reply fitpals/src/components/SearchForm/SearchForm.jsx]: done - removed component

File: fitpals/src/components/Footer/Footer.module.css (hunk 3 above)
Comment (ValentinoMaciej) [reply fitpals/src/components/Footer/Footer.module.css:65]: changed naming convention to proper one

**Review by olga-grochowska — APPROVED** [review]

**Review by Patrex97 — APPROVED** [review]

**Pull request merged**